backend/src/main/kotlin/uk/gov/justice/digital/backend/client/preview/AthenaPreviewClient.kt

```diff
@@ -10,7 +10,7 @@ class AthenaPreviewClient(@Named("preview") private val previewDataSource: DataS
 
     private val logger = LoggerFactory.getLogger(this::class.java)
 
-    override fun runQuery(query: String): List<Map<String, String>> {
+    override fun runQuery(query: String): List<List<String>> {
         logger.info("Executing query: {}", query)
         val startTime = System.currentTimeMillis()
         val statement = previewDataSource.connection.createStatement()
```

Comment:
Question regarding executing the query. 
Do we need any sort of validation of the query or creating a prepared statement if there is a where clause for security reasons?
Currently as I understand we simply execute the query string that exists in the domain correct?

Reply:
That's correct. This is definitely a consideration for the future and ties into query validation at create time. (Currently this is stubbed - originally I'd introduced the spark sql validator which then caused issues with the size of the jar file). I forget the ticket number but there's something on the backlog to address validation.

Reply:
I'll flag with Dom to see if this is acceptable in the short term. If not I'll pick up any work (such as adding a simple C~R~UD keyword check) separately.